src/index.html

```diff
@@ -11,7 +11,114 @@
     <link rel="stylesheet" href="./style.css">
   </head>
   <body>
-    <h1>HTML Form</h1>
     <script type="text/javascript" src="./main.js"></script>
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" >
+      <fieldset>
+        <legend>Personal Information</legend>
+        <div class="personalInformation">
+          <div class="personalInformationField">
+            <label for="surname">
+              Surname:
+            </label>
+            <input type="text" name="surname" id="surname">
+          </div>
```

Comment:
you can remove `<div class="personalInformationField">` wrapper. It's redundant. And refactor each of your input/label connection like this:

```suggestion
            <label class="field">
              Surname:
              <input 
                type="text" 
                name="surname" 
              >
            </label>
```

Apply all over the page